flow/util/genMetrics_21.py

```diff
@@ -0,0 +1,488 @@
+#!/usr/bin/env python3
```

Comment:
remove this file

Reply:
Can we remove this file as part of the next PR which will add the multi-clocks (ETD today).. This file is not being used by any of the scripts. It was used to run standalone for bigDoE that Seungwon was running

Reply:
Removed this file